src/pivot-table/hooks/use-column-width.ts

```diff
@@ -70,6 +71,9 @@ export default function useColumnWidth(
   rect: Rect,
   headersData: HeadersData,
   visibleTopDimensionInfo: VisibleDimensionInfo[],
+  allRowsVisible: boolean,
+  verticalScrollbarWidth: number,
```

Comment:
Not sure if this is a timing thing, isn't this true?` allRowsVisible` -> `verticalScrollbarWidth === 0` or `!allRowsVisible` -> `verticalScrollbarWidth > 0`. So you only need one of them?

Reply:
fixed